shelley/chain-and-ledger/executable-spec/test/Test/Shelley/Spec/Ledger/Rules/TestLedger.hs

```diff
@@ -170,8 +171,8 @@ consumedEqualsProduced =
                 }
               )
         } =
-        (balance u + d + fees + foldl' (+) (Coin 0) rewards)
-          === (balance u' + d' + fees' + foldl' (+) (Coin 0) rewards')
+        (balance u <> (coinToValue $ d + fees + foldl' (+) (Coin 0) rewards))
+          === (balance u'<> (coinToValue $ d' + fees' + foldl' (+) (Coin 0) rewards')) -- TODO what about forging here?
```

Comment:
Seems important

Reply:
this is tricky because this doesn't look at the Tx itself, only compares ledger states - so there is no way to inspect the forge. I made this a strictly Ada comparison

Reply:
This also points out something important: you can no longer sanity-check the total balance of the ledger from the state alone, you need to look at all the historical forges too.